sematic/ui/packages/main/src/components/Metrics.tsx

```diff
@@ -0,0 +1,83 @@
+import { MetricsFilter, useMetrics } from "@sematic/common/src/hooks/metricsHooks";
+import { Alert} from "@mui/material";
+import { useMemo, useState } from "react";
+import {
+    Chart as ChartJS,
+    CategoryScale,
+    LinearScale,
+    PointElement,
+    LineElement,
+    Title,
+    Tooltip,
+    Legend,
+    ChartData,
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+import { MetricPoint } from "@sematic/common/lib/src/Models";
+
+ChartJS.register(
+    CategoryScale,
+    LinearScale,
+    PointElement,
+    LineElement,
+    Title,
+    Tooltip,
+    Legend
+);
+
+export function TimeseriesMetric(props: {
+    metricsFilter: MetricsFilter;
+    color: string;
+    broadcastEvent?: MetricPoint[];
+}) {
+    const { metricsFilter, color, broadcastEvent } = props;
+    const [payload, loading, error] = useMetrics(metricsFilter, broadcastEvent);
```

Comment:
I see a lint error here for `loading`, can you check if you need to handle loading? if not, you can use
`const [payload, , error]` to get rid of the lint error.